src/node.cpp

```diff
@@ -1020,6 +1045,15 @@ bool Node::setparent(Node* p)
     if (parent)
     {
         child_it = parent->children.insert(parent->children.end(), this);
+
+#ifdef ENABLE_SYNC
+        if (parent->type == FILENODE && !isSyncable())
+        {
+            // if child (old version) is not syncable then parent must follow suit (new version)
+            parent->setSyncable(false);
+            setSyncable(true); // set old version back to default
+        }
```

Comment:
This part looks good, but I think we also need to adjust the flags for nodes moved out of a sync, into a sync, or from one sync to another?  thanks

Reply:
I think it would make sense to maintain the syncable property of a node across syncs. If you removed a local file from a sync, then went to say the webclient and moved the corresponding node to another down-sync, it should probably still not be down-synced. Same if you moved the node back to the original sync folder. Thoughts?